app.js

```diff
@@ -0,0 +1,213 @@
+var elId = [
+  document.getElementById('pike'),
+  document.getElementById('seaTac'),
+  document.getElementById('souCen'),
+  document.getElementById('belVue'),
+  document.getElementById('alki')
+];
+
+var hoursOpen = [
+  '10:00a',
+  '11:00a',
+  '12:00p',
+  '01:00p',
+  '02:00p',
+  '03:00p',
+  '04:00p',
+  '05:00p',
+  '06:00p'
+];
+
+var stores = [
+  pike,
+  // sea,
+  // south,
+  // bell,
+  // alki
+];
+
+var pike = {
+  name: 'Pike Place',
+  minCust: 17,
+  maxCust: 88,
+  avgSale: 5.2,
+  hrTotal: [],
+  dayTotal: 0,
+  id: elId[0],
+
+  randCust: function(min, max) {
+    return Math.random() * (max - min + 1) + min;
+  },
+
+  genHr: function(cookieHr, total, avg, rand, hours) {
+    for (var i = 0; i < hours.length; i++) {
+      var hrCook = Math.floor(rand * avg);
```

Comment:
Your logic is right here, but within your `Math.floor()` you should call `randCust()` and multiply by `this.avgSale`